steganography.py

```diff
@@ -151,5 +286,51 @@ def unmerge(img, output):
     unmerged_image.save(output)
 
 
+@cli.command()
+@click.option('--img', required=True, type=str, help='Image to insert hex code into')
+@click.option('--hex_code', required=True, type=str, help='Hex code to be inserted into the image')
+@click.option('--output', required=True, type=str, help='Output Image')
+@click.option('--x', required=False, type=int, default=0, help='X Co-ordinate for pixel that hex_insertion beings with.'
+                                                               ' Defaults to 0.')
+@click.option('--y', required=False, type=int, default=0, help='Y Co-ordinate for pixel that hex_insertion begins with.'
+                                                               ' Defaults to 0.')
+@click.option('--rgb', required=False, type=str,
+              help='2D JSON Array (3x8) of [1/0] values to indicate which RGB values should be overwritten. '
+                   'Defaults to the four LSBs of each RGB value.',
+              default="[[0,0,0,0,1,1,1,1],[0,0,0,0,1,1,1,1],[0,0,0,0,1,1,1,1]]")
+# Default RGB values to be overwritten:
+#        0  1  2  3  4  5  6  7
+#       _______________________
+# R)  0 |0  0  0  0  1  1  1  1
+# G)  1 |0  0  0  0  1  1  1  1
+# B)  2 |0  0  0  0  1  1  1  1
```

Comment:
I believe it would be more readable to add these comments as docstrings, for example:

```python
def insert_hex_code(img, hex_code, output, x, y, rgb):
    """
    Default RGB values to be overwritten:
            0  1  2  3  4  5  6  7
           _______________________
    R)  0 | 0  0  0  0  1  1  1  1
    G)  1 | 0  0  0  0  1  1  1  1
    B)  2 | 0  0  0  0  1  1  1  1
    """
    hexed_image = Steganography.insert_hex(Image.open(img), hex_code, x, y, rgb)
    hexed_image.save(output)
```

What do you think?